cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -1130,6 +1130,8 @@
     public static final String CFG_CDAP_MASTER_KRB_PRINCIPAL = "cdap.master.kerberos.principal";
 
     public static final String UGI_CACHE_EXPIRATION_MS = "cdap.ugi.cache.expiration.ms";
+    
+    public static final String CDAP_RANGER_LDAP_USESYNC_CASECENVERSION = "cdap-ranger.usersync.ldap.username.caseconversion";
```

Comment:
Configuration should be renamed to something like 'security.authentication.username.caseconversion'

Reply:
Updated config name